packages/discovery-react-components/src/utils/__tests__/useDataApi.test.tsx

```diff
@@ -58,7 +58,11 @@ class AggregationResultSearchClient extends BaseSearchClient {
 
 class ErrorSearchClient extends BaseSearchClient {
   public async query(): Promise<any> {
-    return Promise.reject();
+    return Promise.reject(
+      new Error(
+        '"status_code":400,"errors":[{"code":"query_daily_limit","message":"You have exceeded the number of daily queries allowed for your plan. You can resume requests after the daily reset.","more_info":"https://cloud.ibm.com/docs/discovery-data"}],"trace":"TOOLING-9947g26q0x1x9"}'
```

Comment:
is this how the error is formatted? I think it might look something more like
```typescript
const error = new Error();
error.message = 'You have exceeded the number of daily queries allowed for your plan. You can resume requests after the daily reset.'
error.body = {status_code: 400, errors: [{code:'query_daily_limit',message: 'You have exceeded the number of daily queries allowed for your plan. You can resume requests after the daily reset.'}]};
```